Doppler.ContactPolicies.Api/Controllers/ContactPolicies.cs

```diff
@@ -36,9 +36,8 @@ public async Task<IActionResult> GetContactPoliciesSettings(string accountName)
         public async Task<IActionResult> UpdateContactPoliciesSettings(string accountName,
                     [FromBody] ContactPoliciesSettingsDto contactPoliciesSettings)
         {
-            // TODO: replace this validation to get UserId by accountName
-            var currentContactPoliciesSetting = await _contactPoliciesService.GetContactPoliciesSettingsAsync(accountName);
-            if (currentContactPoliciesSetting == null)
+            var userFounded = await _contactPoliciesService.GetIdUserByAccountName(accountName);
+            if (userFounded == null)
```

Comment:
In some places you are using `is null` for _null_ comparison and in other places, you are using `== null`.

Could you use the same way always? or is there a reason?

Reply:
Resolved!